mikrotik-backup/backup.sh

```diff
@@ -24,13 +24,13 @@ do
 	echo Backing up "$router"... 1>&2
 	SSH_CMD="ssh ${SSH_OPTS} $router"
 
+	# ROS6 and ROS7 have a different date output.  Need to accept either here.
+	# Old pattern: 's![a-z]*/[0-3][0-9]/20[0-9][0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9]!mm/dd/yyyy hh:mm:ss!'
+
 	$SSH_CMD '/export hide-sensitive' \
-	| sed 's![a-z]*/[0-3][0-9]/20[0-9][0-9] [0-2][0-9]:[0-5][0-9]:[0-5][0-9]!mm/dd/yyyy hh:mm:ss!' \
+	| sed 's![0-9a-z/-][0-9a-z/-]* [0-2][0-9]:[0-5][0-9]:[0-5][0-9]!mm/dd/yyyy hh:mm:ss!' \
```

Comment:
Rather than commenting the old pattern, it would be helpful to provide examples of the two formats the pattern needs to capture.

Reply:
Done.